src/test/java/core/basesyntax/strategy/impl/PurchaseOperationHandlerTest.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.storage.Storage;
+import org.junit.Before;
+import org.junit.Test;
+
+public class PurchaseOperationHandlerTest {
+    private Fruit fruit;
+    private PurchaseOperationHandler purchaseOperationHandler;
+    private FruitTransaction fruitTransaction;
+
+    @Before
+    public void setUp() {
+        purchaseOperationHandler = new PurchaseOperationHandler();
+        Storage.storage.put(new Fruit("banana"), 88);
+        fruit = new Fruit("banana");
+        fruitTransaction = new FruitTransaction("p", fruit, 89);
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void purchase89_BananaFromEmptyStorage_NotOk() {
```

Comment:
I think it's better not to use numbers in the names of test methods! Better focus on the result of the test!

Reply:
I think that this way our test will be more informative in the title